<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shv::visu::logview::DlgLogInspector</class>
 <widget class="QDialog" name="shv::visu::logview::DlgLogInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>988</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblInfo">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&amp;Since</string>
          </property>
          <property name="buddy">
           <cstring>edSince</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="edSince">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="specialValueText">
           <string>Not defined</string>
          </property>
          <property name="minimumDate">
           <date>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd hh:mm:ss</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&amp;Until</string>
          </property>
          <property name="buddy">
           <cstring>edUntil</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateTimeEdit" name="edUntil">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="specialValueText">
           <string>Not defined</string>
          </property>
          <property name="minimumDate">
           <date>
            <year>2000</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yyyy-MM-dd hh:mm:ss</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btMoreOptions">
          <property name="text">
           <string>&amp;Options</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btLoad">
          <property name="text">
           <string>Download</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Options</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <property name="spacing">
          <number>5</number>
         </property>
         <item row="0" column="1" colspan="4">
          <widget class="QLineEdit" name="edShvPath"/>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="edTimeout">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="suffix">
            <string> sec</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="edMaxRecordCount">
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="singleStep">
            <number>1000</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Max record count</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="chkUseHistoryProvider">
           <property name="text">
            <string>use &amp;history provider</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Timeout</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QLineEdit" name="edPathPattern"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Path pa&amp;ttern</string>
           </property>
           <property name="buddy">
            <cstring>edPathPattern</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="5">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>SHV &amp;Path</string>
           </property>
           <property name="buddy">
            <cstring>edShvPath</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="chkPathsDict">
           <property name="text">
            <string>paths dict</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="chkWithSnapshot">
           <property name="text">
            <string>with snapshot</string>
           </property>
          </widget>
         </item>
         <item row="1" column="5">
          <widget class="QCheckBox" name="chkIsPathPatternRexex">
           <property name="toolTip">
            <string>Path pattern is regex</string>
           </property>
           <property name="text">
            <string>&amp;regex</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGraph">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="shv::visu::timeline::GraphView" name="graphView">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>930</width>
            <height>218</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabData">
      <attribute name="title">
       <string>Data</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="edFilter">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btSaveData">
           <property name="text">
            <string>Save data</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tblData"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabInfo">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPlainTextEdit" name="edInfo"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>shv::visu::timeline::GraphView</class>
   <extends>QScrollArea</extends>
   <header>src/timeline/graphview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edSince</tabstop>
  <tabstop>edUntil</tabstop>
  <tabstop>btMoreOptions</tabstop>
  <tabstop>edShvPath</tabstop>
  <tabstop>edPathPattern</tabstop>
  <tabstop>chkIsPathPatternRexex</tabstop>
  <tabstop>edMaxRecordCount</tabstop>
  <tabstop>chkPathsDict</tabstop>
  <tabstop>chkUseHistoryProvider</tabstop>
  <tabstop>edTimeout</tabstop>
  <tabstop>chkWithSnapshot</tabstop>
  <tabstop>btLoad</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>graphView</tabstop>
  <tabstop>tblData</tabstop>
  <tabstop>edInfo</tabstop>
  <tabstop>edFilter</tabstop>
  <tabstop>btSaveData</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btMoreOptions</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_3</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>639</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
